kDrive/UI/View/Files/Search/TextInputTableViewCell.swift

```diff
@@ -0,0 +1,58 @@
+/*
+ Infomaniak kDrive - iOS App
+ Copyright (C) 2023 Infomaniak Network SA
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import Foundation
+import InfomaniakCoreUI
+import kDriveCore
+import kDriveResources
+import MaterialOutlinedTextField
+import UIKit
+
+final class TextInputTableViewCell: UITableViewCell {
+    @IBOutlet var textField: MaterialOutlinedTextField!
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+
+        textField.setInfomaniakColors()
+        textField.setHint(KDriveResourcesStrings.Localizable.sortExtension)
+        textField.placeholder = ".jpg, .mov …"
```

Comment:
```suggestion
        textField.placeholder = ".jpg, .mov, etc."
```
To be iso with the webapp 😊

Reply:
discussed, I keep it this way.